common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3078,4 +3083,17 @@ object CelebornConf extends Logging {
       .doc("Threads count for read buffer per mount point.")
       .intConf
       .createWithDefault(8)
+
+  val METRICS_EXTRA_LABELS: OptionalConfigEntry[Seq[String]] =
```

Comment:
we don't need `Optional` here, just simply assign `Seq.empty` as the default value.